proto/v1/database_service.proto

```diff
@@ -244,6 +251,18 @@ message GetDatabaseSchemaRequest {
   bool sdl_format = 2;
 }
 
+message GetDatabaseSchemaDiffRequest {
+  // The name of the database to retrieve schema.
+  // Format: instances/{instance}/databases/{database}/schema
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Might be a schema string too? One of our users want to get a diff between special versions of the target database, not always the latest version.

Reply:
What about using "instances/{instance}/databases/{database}/changeHistories/latest" And "instances/{instance}/databases/{database}/changeHistories/{change}"? For this use case? Either using change history as the name field or adding additional optional field.

Reply:
SGTM